utils/utils.h

```diff
@@ -574,10 +575,13 @@ typedef struct gbConfig {
   bool isDynamic;
   char *GB_LOG_LEVEL;
   ssize_t GB_GLFS_LRU_COUNT;
+  ssize_t GB_CLI_TIMEOUT;  /* seconds */
 } gbConfig;
 
 int glusterBlockSetLogLevel(unsigned int logLevel);
 
+//int glusterBlockSetCliTimeout(size_t timeout);
+
```

Comment:
any reason to keep the commented code? Plan for further changes? OK with removing this too.

Reply:
At the definition of this function, I have marked a TODO, it goes like this:

When we actually want to use gbConf (global configuration buffer) in cli process this will be used. I have plans to work on it in the near future, so just preserving it here.

Thanks!